ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -459,3 +459,63 @@ if get_option('enable-mediapipe')
     install_dir: filter_subplugin_install_dir
   )
 endif
+
+if get_option('enable-snpe')
+  if host_machine.system() != 'linux' and host_machine.cpu_family() != 'x86_64'
```

Comment:
Q: does SNPE support only linux x86_64?

Reply:
No, SNPE also supports arm architecture and Android. But the their development environment is limited Ubuntu.

Reply:
@helloahn @myungjoo I updated the meson.build not to restrict the host_machine cause SNPE SDK also provides libraries for other host.
I'll deal with this in later PRs.